napari_pyclesperanto_assistant/_gui/_category_widget.py

```diff
@@ -140,7 +140,7 @@ def _generate_signature_for_category(category: Category) -> Signature:
         Parameter(f"input{n}", k, annotation=t) for n, t in enumerate(category.inputs)
     ]
     # Add valid operations choices (will create the combo box)
-    choices = list(cle.operations(category.include, category.exclude))
+    choices = list(cle.operations(['in assistant'] + list(category.include), category.exclude))
```

Comment:
ah yes!  I had that in an earlier "make_enum" implementation and forgot to add it back.  thanks!